src/libraries/Microsoft.Extensions.Primitives/src/StringSegmentComparer.cs

```diff
@@ -33,6 +42,12 @@ public bool Equals(StringSegment x, StringSegment y)
             return StringSegment.Equals(x, y, Comparison);
         }
 
+        /// <summary>
```

Comment:
>summary [](http://example.com/codeflow?start=13&length=7)

this missing the parameter doc element.

Reply:
Whoops. Missed that this `GetHashCode()` had a parameter. Pushing a change to resolve this, with the context changed